### DISCUSSION TITLE
2005/08/11/%E8%AE%B0%E6%95%B0%E5%99%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#44] [Announcements]
# 2005/08/11/%E8%AE%B0%E6%95%B0%E5%99%A8

本页一共被访问了：

https://blog.qizhen.xyz/2005/08/11/%E8%AE%B0%E6%95%B0%E5%99%A8

**Comment by ruanqizhen** [comment]
JING
2005年08月13日 18:52 编辑
我是学医的,我一个同学双学位修的是计算机,他跟我说,医学跟计算机合在一起,会害死人的!!看来站长是强人中的强人啊!!

回复
OLIVIA
2005年08月12日 14:49 编辑
好专业~~~

回复

**Comment by ruanqizhen** [comment]
这个二十年前的计数器居然还在工作！
